src/SimpleSAML/Error/Error.php

```diff
@@ -206,6 +206,25 @@ protected function saveError(): array
         } else {
             $referer = 'unknown';
         }
+
+        $showerrors = $config->getOptionalBoolean('showerrors', true);
+
+        $whitelist = Configuration::getInstance()->getOptionalArray('showerrors.whitelist', ['*' => true]);
```

Comment:
In terms of the backward compatibility, if the new configuration of 'showerrors.whitelist' is missing from the exisiting configuration file, it will not filter out any error as default?

Reply:
I have made the initial version of the PR not change behaviour by default. If this PR seems like a good thing to pursue  we can create a curated list of errors to hide and add that as a breaking change at some version.